### ISSUE TITLE
Limit Available Assignments to next four weeks

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#4869]
Per today's conversation with Lewis, want to limit what officers can sign up for through the next four weeks. This makes the "All", "Next Week", "Next Month" filters kind of useless, so we'll simplify the interface a bit by removing them. 

<img width="329" alt="no-filters" src="https://user-images.githubusercontent.com/1463708/104308748-c319af00-5496-11eb-8501-6f17af1231cc.png">

**Comment by amenity** [comment]
@SurbhiBakshi I think it also makes sense to remove the "All" "Next Week" "Next Month" button filters. What do you think?

**Comment by SurbhiBakshi** [comment]
@amenity - since we will only show the next 4 weeks, yes, agree those filters can go

**Comment by amenity** [comment]
@mddilley - I think we'll need your help on this. And as I'm testing on mobile, I see that hiding those date filters will really simplify the home screen and save space. 
<img width="3
<img width="329" alt="no-filters" src="https://user-images.githubusercontent.com/1463708/104308748-c319af00-5496-11eb-8501-6f17af1231cc.png">

**Comment by amenity** [comment]
@mddilley it would be great to take care of this in Sprint 42 if possible.

**Comment by mddilley** [comment]
Sounds great to me. I'll start on this today. 👍

**Comment by mddilley** [comment]
The code is now updated in [this PR](https://github.com/cityofaustin/atd-knack/pull/17) to remove the time filter UI and also filter the assignment records for the "Available Assignments" view to those that fall in during the next four weeks. These updates are live in the test instance called `Test - Vision Zero in Action | 8 Jan 2021` and can be added to production when reviewed.

### Updated UI
![Screen Shot 2021-01-13 at 5.46.17 PM.png](https://images.zenhubusercontent.com/5cfe7d078509cb6dcbaafdc3/c0908186-f0aa-4df2-b94e-58610649fb42)

**Comment by amenity** [comment]
Looks & works great, @mddilley! Thanks.

**Comment by SurbhiBakshi** [comment]
Thanks @mddilley! I added more assignments to the Test instance and the filter for the next four weeks works great!

**Comment by amenity** [comment]
We're ready for this to go live, @mddilley 💪 🙏.

**Comment by mddilley** [comment]
TIL that in Knack, "is during the next 4 weeks" means "during the next 4 weeks starting on the next upcoming Sunday" and does not start on the current day. ⌚🙃  

I updated the query for the default assignments to return any assignments that are today or in the next 28 days. This returns assignments scheduled for today, January 15th, through Friday, February 12th. This has been updated in the repo and in production.
